templates/mip/app.yaml

```diff
@@ -0,0 +1,5 @@
+# This manifest holds the information the app needs to run on the Nextmv Cloud.
+type: sdk
+features:
+  - sdk
+  - mip-highs # Replace with mip-xpress to use the mip.Xpress solver provider.
```

Comment:
Should we already advertise this here? We would need to update this too, if we introduce further solvers. :thinking:

Reply:
I'll add a more generic comment.